sonar-php-plugin/src/main/java/org/sonar/plugins/php/cache/ProjectSymbolDataSerializer.java

```diff
@@ -0,0 +1,131 @@
+package org.sonar.plugins.php.cache;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.sonar.api.internal.apachecommons.io.IOUtils;
+import org.sonar.php.symbols.ClassSymbolData;
+import org.sonar.php.symbols.FunctionSymbolData;
+import org.sonar.php.symbols.MethodSymbolData;
+import org.sonar.php.symbols.Parameter;
+import org.sonar.php.symbols.ProjectSymbolData;
+import org.sonar.plugins.php.api.symbols.QualifiedName;
+import org.sonar.plugins.php.api.visitors.LocationInFile;
+
+public class ProjectSymbolDataSerializer {
+
+  private final ByteArrayOutputStream stream;
+  private final VarLengthOutputStream out;
+  private final StringTable stringTable;
+
+  private ProjectSymbolDataSerializer() {
+    stream = new ByteArrayOutputStream();
+    out = new VarLengthOutputStream(stream);
+    stringTable = new StringTable();
```

Comment:
The StringTable is still unused

Reply:
It's used here: https://github.com/SonarSource/sonar-php/pull/1006/files#diff-e85bd0775b076fa5ed2f453cb58086c6893499efcd10bfe0a1001d19c07c3627R120-R122

Reply:
Yeah, but only the index of a string in the table is used. So far I understood, we also have to store the table as well, don't we?